pooch/tests/test_core.py

```diff
@@ -45,6 +46,11 @@
 }
 
 
+@pytest.fixture
+def pooch_tmp_path(tmp_path):
```

Comment:
```suggestion
def pooch_tmp_path(tmp_path):
    """
    Mirror the test data folder on a temporary directory. Needed to avoid
    permission errors when pooch is installed on a non-writable path.
    """
```